draft-ietf-scitt-architecture.md

```diff
@@ -680,31 +687,31 @@ Editor's Note: This may be moved to appendix.
 
 All messages are sent as HTTP GET or POST requests.
 
-If the transparency service cannot process a client's request, it MUST return an HTTP 4xx or 5xx status code, and the body SHOULD be a JSON problem details object ({{RFC7807}}) containing:
+If the Transparency Service cannot process a client's request, it MUST return an HTTP 4xx or 5xx status code, and the body SHOULD be a JSON problem details object ({{RFC7807}}) containing:
 
 - type: A URI reference identifying the problem.
 To facilitate automated response to errors, this document defines a set of standard tokens for use in the type field within the URN namespace of: "urn:ietf:params:scitt:error:".
 
-- detail: A human-readable string describing the error that prevented the transparency service from processing the request, ideally with sufficient detail to enable the error to be rectified.
+- detail: A human-readable string describing the error that prevented the Transparency Service from processing the request, ideally with sufficient detail to enable the error to be rectified.
 
 Error responses SHOULD be sent with the `Content-Type: application/problem+json` HTTP header.
```

Comment:
why not `application/json`....

Reply:
Out of scope for this PR I would say. Maybe open an issue to discuss it?

Reply:
yeah, def out of scope, was just wondering if this had been discussed, I will file an issue.

Reply:
https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/issues/62